### DISCUSSION TITLE
Nuke config and re-onboard not working

### DISCUSSION THREAD

**Discussion by andrewscfc** [renovatebot/renovate#8310] [Request Help]
Hi,
 
We’ve recently activated renovate on our open source repo and we chose to close our onboarding PR. We are trying to get a new one recreated following the instructions here: https://docs.renovatebot.com/reconfigure-renovate/#nuke-config-and-re-onboard
 
Here is our old onboarding PR we have renamed as per the instructions: https://github.com/bbc/simorgh/pull/8710
 
Here is our repo: https://github.com/bbc/simorgh

(I've raised this via support email too but anxious to get going with renovate! case number #00061103 )

Thanks
Andrew

**Comment by viceice** [comment]
Maybe you need to disable / reenable the app. The app will reduce crawling rate of disabled repos.

**Comment by rarkins** [comment]
I should be able to trigger it to run again

**Reply by rarkins** [reply]
@andrewscfc The [`renovate` branch](https://github.com/bbc/simorgh/tree/renovate) in this repo prevents a `renovate/configure` branch from being created. Can you delete that? If a branch like `foo` exists then you cannot create any branches like `foo/bar`

**Reply by andrewscfc** [reply]
Hey @rarkins , I've deleted the branch now, thanks for looking at this @rarkins 🙂

**Reply by andrewscfc** [reply]
That has now worked @rarkins, thank you!

**Reply by rarkins** [reply]
Great. Can you also let support know they can close the ticket?

**Reply by andrewscfc** [reply]
sure, I will when they get back to me (I only contacted them this morning)
